examples/github-app/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "github-app",
+  "version": "0.1.0",
```

Comment:
this should probably have `"private": true`, so it's not published by accident.

Reply:
Line 21 has that

Reply:
ah ok, buried in there :-)

Reply:
Can't make it easy or it wouldn't be as fun 😆